### DISCUSSION TITLE
list of supported modules

### DISCUSSION THREAD

**Discussion by teoremakotelnikova** [ni/nimi-python#1675] [Q&A] [answered]
hi!
Is there a list of modules supported available? 
can't find =(
ex. could I use nidigital for 6555?

**Comment by marcoskirsch** [comment, marked as answer]
All models supported by the corresponding driver runtimes are supported. For the list of NI-Digital Pattern Driver supported hardware, please see [its Readme file](https://www.ni.com/pdf/manuals/376615k.html#supported).

The NI PXIe-6555 is programmed using NI-HSDIO. There is no official Python API for NI-HSDIO. Sorry!
You can use `nimi-python` as a starting point if you decide it’s worth your while to write your own Python bindings for NI-HSDIO.
